### PR TITLE
Add missing hooks

### PR DESCRIPTION
**Pull request by Hlavtox**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | There are several issues when there is an alias for a hook, but this module is not present in the database. It should be fixed in the core probably, but this mitigates the issue. Related to https://github.com/PrestaShop/PrestaShop/pull/35215.
| Type?             | refactor
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | 
| Sponsor company   | 
| How to test?      | Run these queries in phpmyadmin, replace PREFIX_hook with your prefix.

### CODE REVIEW

**Review by kpodemski — APPROVED** [review]

**Comment by kpodemski** [comment]
Blocked until the PR from the core will be merged.

**Comment by Hlavtox** [comment]
@kpodemski Ready to go. :-)

**Comment by ShaiMagal** [comment]
LGTM, tested the query and working properly. Duplicate hooks are ignored. ✅

Snímek obrazovky 2024-01-30 163622

**Comment by Quetzacoalt91** [comment]
Related to:
* https://github.com/PrestaShop/PrestaShop/pull/34133

**Pull request merged**
